src/custom-providers/azext_custom_providers/tests/latest/test_custom-providers_scenario.py

```diff
@@ -0,0 +1,82 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+class CustomProvidersScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_custom_providers')
+    def test_custom_providers_common_scenario(self):
+
+        rp_name = self.create_random_name('clitest-crp', 20)
+        self.kwargs.update({
+            'rp_name': rp_name
+        })
+
+        self.cmd('az custom-providers resource-provider create '
+                 '--resource-group {rg} '
+                 '--name {rp_name} '
+                 '--location westus2 '
+                 '--action '
```

Comment:
can we add more than one `action` in the test?

Reply:
yes, I'll add more actions.

---
In reply to: [422454899](https://github.com/Azure/azure-cli-extensions/pull/1627#discussion_r422454899) [](ancestors = 422454899)